content/en/Integrations/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Cobalt Integrations"
+linkTitle: "Integrations"
+weight: 110
+description: >
+  Integrate third-party apps and configure webhooks.
+---
+
+{{% pageinfo %}}
+Streamline your pentesting and development workflows with Cobalt integrations.
+{{% /pageinfo %}}
+
+{{% alert title="Note" color="note" %}}
+The availability of this feature is based on your <a href="https://www.cobalt.io/pentest-pricing" target="_blank">PtaaS tier</a>.
+{{% /alert %}}
+ 
+Navigate to the **Integrations** page to get started.
```

Comment:
```suggestion
In the Cobalt app, navigate to the **Integrations** page to get started.
```
@mjang 
My assumption:
In Product Docs, we describe the user experience in the Cobalt app. We only have one app. We assume that the user has already signed in and is exploring the app features. That's why I'm only adding "in the Cobalt app." Maybe it's my habit from my previous projects. 
For API use cases, of course we include the steps for working with the API.